opennms-base-assembly/src/main/filtered/etc/events/opennms.internal.events.xml

```diff
@@ -248,10 +248,25 @@
    <event>
       <uei>uei.opennms.org/internal/applicationDeleted</uei>
       <event-label>OpenNMS-defined application event: applicationDeleted</event-label>
-      <descr>&lt;p>Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.&lt;/p></descr>
-      <logmsg dest="logndisplay">Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.</logmsg>
+      <descr>Application '%parm[applicationName]%' with ID '%parm[applicationId]%' has been deleted.</descr>
+      <logmsg dest="logndisplay">Application '%parm[applicationName]%' has been deleted.</logmsg>
       <severity>Warning</severity>
    </event>
+   <event>
+      <uei>uei.opennms.org/internal/applicationChanged</uei>
+      <event-label>OpenNMS-defined node event: applicationChanged</event-label>
+      <descr>The application '%parm[applicationName]%' with ID %parm[applicationId]% has been changed.</descr>
```

Comment:
I would single-quote %parm[applicationId]% here for consistency with the existing events